pygmt/tests/test_figure.py

```diff
@@ -292,6 +292,33 @@ def mock_psconvert(*args, **kwargs):  # pylint: disable=unused-argument
     }
 
 
+@pytest.mark.parametrize("fmt", [".png", ".pdf", ".jpg", ".tif"])
```

Comment:
Test `bmp`, and sort alphabetically. Edit to add `eps` when you figure out if an `esw` file can be created.
```suggestion
@pytest.mark.parametrize("fmt", ["bmp", ".jpg", ".pdf", ".png", ".tif"])
```